de.dlr.sc.virsat.model.extension.statemachines.ui/src/de/dlr/sc/virsat/model/extension/statemachines/ui/diagram/features/StateMachineDiagramFeatureProvider.java

```diff
@@ -110,18 +113,27 @@ public IAddFeature getAddFeature(IAddContext context) {
 	public ICustomFeature[] getCustomFeatures(ICustomContext context) {
 		PictogramElement[] pe = context.getPictogramElements();
 		Object object = getBusinessObjectForPictogramElement(pe[0]);
-		if (object instanceof State) {
+		List<ICustomFeature> customFeaturs = new ArrayList<>();

```

Comment:
Rename `customFeaturs` to `customFeatures`?

Reply:
Nice catch!